benchmark-serve.sh

```
@@ -0,0 +1,42 @@
+start_time=$(date '+%Y-%m-%d-%H:%M:%S')
+output_folder=benchmark.run.${SLURM_JOB_ID}.$(date '+%Y-%m-%d-%H:%M:%S')
+
+run_benchmark() {
+    bash ~/bin/dev-tekit-d.sh ./run_benchmark_serve.sh ${output_folder}
+}
+
+report() {
+    results=$1
+    echo "Performance report ${SLURM_JOB_ID} - trtllm-serve"
+    echo "==========================================="
+    echo "Report path" $(realpath ${results})
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)